### PR TITLE
replace references to mapboxgl global with maplibregl

### PR DESCRIPTION
**Pull request by rotu**

### CODE REVIEW

File: docs/bin/create-image.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ body { margin:0; padding:0; }
 </head>
 <body>
 <script>
-mapboxgl.accessToken = '${token}';
+maplibregl.accessToken = '${token}';
```
Comment (HarelM) [inline docs/bin/create-image.ts:46]: There's no token in maplibre, it was removed.
This script is probably broken if I need to guess...

File: docs/pages/api/index.md [hunk 2]
```diff
@@ -30,28 +30,25 @@ overviewHeader:
 
 MapLibre GL JS is a JavaScript library that uses WebGL to render interactive maps from vector tiles and [MapLibre styles](https://maplibre.org/maplibre-gl-js-docs/style-spec/). It is part of the [MapLibre GL ecosystem](https://github.com/maplibre).
 
-
 ## Quickstart
 
 {{
-    <Example
```
Comment (HarelM) [inline docs/pages/api/index.md:37]: Why was the indentation removed?

Comment (rotu) [reply docs/pages/api/index.md]: The perils of a markdown plugin in vscode

File: docs/pages/api/index.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ For strict CSP environments without `worker-src blob: ; child-src blob:` enabled
 {{
 <Copyable lang="html">{`<script src='${urls.js().replace('.js', '-csp.js')}'></script>
 <script>
-mapboxgl.workerUrl = "${urls.js().replace('.js', '-csp-worker.js')}";
+maplibregl.workerUrl = "${urls.js().replace('.js', '-csp-worker.js')}";
```
Comment (HarelM) [inline docs/pages/api/index.md:92]: There's also another two references just above this line

Comment (HarelM) [reply docs/pages/api/index.md:92]: There's also something about mapbox token below this part, which I also think can be removed.

**Comment by HarelM** [comment]
@rotu can you fix the CR comments so I can merge?

**Comment by rotu** [comment]
Nothing left here to merge.

**Pull request closed without merging**